stdlib/2.7/ConfigParser.pyi

```diff
@@ -0,0 +1,140 @@
+from typing import Any, Tuple
+
+__all__ = None # type: list[str]
+DEFAULTSECT = None # type: str
+MAX_INTERPOLATION_DEPTH = None # type: int
+
+class Error(Exception):
+    def _get_message(self) -> None: ...
+
```

Comment:
Please disregard what your linter may tell you -- we prefer no blank lines between methods in stubs.